modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -82,22 +82,12 @@ you use the Preview version of `SidePanel` until the one in Main is replaced wit
 
 ---
 
-### Layout and Column
+### Stack, HStack and vstack
 
-`Layout` and `Column` components have been [hard-deprecated](#hard-deprecation) and are no longer
-available in Canvas Kit. Please consider using
-[`Grid`](https://workday.github.io/canvas-kit/?path=/docs/components-containers-grid--basic)
-instead. While the `Grid` component is not a 1:1 replacement for `Layout`, it can be used in the
-same way and offers several benefits over the deprecated components as a more robust and flexible
-layout solution.
-
-Please refer to
-[the Examples section](https://workday.github.io/canvas-kit/?path=/docs/examples-layouts--custom-column-positioning)
-for ideas on how to replace the deprecated `Layout` component with the `Grid` component. This will
-provide you with a deeper understanding of how to set custom column width or positioning using the
-`Grid` component.
-
----
+We've hard deprecated the `Stack`, `HStack` and `VStack` components along with it's props
+(`StackProps`, `HStackProps`, `VStackProps`, `StackStyleProps`). We recommend using the `Flex`
+component and replace the `spacing` prop with the `gap`. You can also run
+`npx @workday/canvas-kit-codemod softDeprecate/Stack`.
```

Comment:
If this is codemodable, we should add the robot emoji, but we can do that before we launch to clean up the docs for it, just making a note. here

Reply:
Definitely. I'll leave this here as a reminder